massa-deferred-calls/src/lib.rs

```diff
@@ -554,11 +548,7 @@ impl DeferredCallRegistry {
                 .deserialize::<DeserializeError>(serialized_value)
                 .is_ok();
         } else if serialized_key.eq(DEFERRED_CALL_TOTAL_REGISTERED.as_bytes()) {
-            return self
-                .registry_changes_deserializer
-                .total_calls_registered_deserializer
-                .deserialize::<DeserializeError>(serialized_value)
-                .is_ok();
+            return true;
```

Comment:
maybe directly delete the entry here if it is present and create a followup issue to remove this check and the DEFERRED_CALL_TOTAL_REGISTERED constant completely from the code